docs/user-guide/knative.md

```diff
@@ -0,0 +1,55 @@
+# Knative Integration
+
+[Knative](https://knative.dev/) is is a popular Kubernetes-based platform for managing serverless workloads.
+
+Replacing Istio with Ambassador can potentially reduce the operational overhead of running Knative. Ambassador can watch for changes in Knative configuration in your Kubernetes cluster and set up routing accordingly.
+
+## Getting started
+
+#### Prerequisites
+
+Knative requires a Kubernetes cluster v1.11 or newer with the MutatingAdmissionWebhook admission controller enabled. kubectl v1.10 is also required. This guide assumes that you’ve already created a Kubernetes cluster which you’re comfortable installing alpha software on.
+
+1. Install and configure Knative:
+
+   ```
+   kubectl apply -f https://github.com/knative/serving/releases/download/v0.7.1/serving.yaml
+   ```
+   
+   By default, Knative uses Istio of all ingress traffic. After applying the above manifests, edit the `config-network` ConfigMap to use Ambassador:
```

Comment:
You actually don't need to patch the `config-network` ConfigMap to get this working anymore.

Reply:
fixed